File: src/main/java/org/codingkataos/refactoring/move/Book.java
package org.codingkataos.refactoring.move;

public class Book {
    private String name;
    private String author;
    private String content;

    public boolean contains(String word){
        return content.contains(word);
    }

    public String getAuthor(){
        return author;
    }

    public String exportToPDF(){
        return convertToPDF();
    }

    public String getName() {
        return name;
    }

    public void sendContentTo(String email){
        // send content to email.
    }

    private String convertToPDF(){
        return content;
    }
}

File: csharp/refactoring/Move/Book.cs
namespace Refactoring.move
{

	public class Book
	{
		private string name;
		private string author;
		private string content;

		public bool Contains(string word)
		{
			return content.Contains(word);
		}

		public string Author
		{
			get{ return author; }
		}

		public string ExportToPDF()
		{
			return ConvertToPDF();
		}

		public string Name
		{
			get { return name; }
		}

		public void SendContentTo(string email)
		{
			// send content to email.
		}

		private string ConvertToPDF()
		{
			return content;
		}
	}
}